Description: use system libs
 make use of sytems libs, instead of embedded ones
Author: Nicolas HOUDELOT <nicolas@demosdebs.org>
Applied-Upstream: no
Last-Update: 2024-03-10
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/XArchive/3rdparty/bzip2/bzip2.pri
+++ b/XArchive/3rdparty/bzip2/bzip2.pri
@@ -8,7 +8,7 @@
     LIBS += $$PWD/libs/bzip2-win-$${QT_ARCH}.lib
 }
 unix:!macx {
-    LIBS += $$PWD/libs/libbzip2-unix-$${QT_ARCH}.a
+    LIBS += -lbz2
 }
 unix:macx {
     LIBS += $$PWD/libs/libbzip2-macos-$${QT_ARCH}.a
--- a/XArchive/xarchive.pri
+++ b/XArchive/xarchive.pri
@@ -39,12 +39,12 @@
 
 !contains(XCONFIG, zlib) {
     XCONFIG += zlib
-    include($$PWD/3rdparty/zlib/zlib.pri)
+    LIBS += -lz
 }
 
 !contains(XCONFIG, bzip2) {
     XCONFIG += bzip2
-    include($$PWD/3rdparty/bzip2/bzip2.pri)
+    LIBS += -lbz2
 }
 
 !contains(XCONFIG, lzma) {
--- a/XQwt/xqwt.pri
+++ b/XQwt/xqwt.pri
@@ -3,7 +3,8 @@
 
 !contains(XCONFIG, qwt) {
     XCONFIG += qwt
-    include($$PWD/3rdparty/qwt/qwt.pri)
+    INCLUDEPATH += /usr/include/qwt/
+	LIBS += -lqwt-qt5
 }
 
 DISTFILES += \
